sys/include/net/eui48.h

```diff
@@ -56,6 +56,22 @@ static inline void eui48_to_eui64(eui64_t *eui64, const eui48_t *addr)
     eui64->uint8[7] = addr->uint8[5];
 }
 
+/**
+ * @name EUI-48 bit flags contained in the first octet
+ * @{
```

Comment:
Can you add the reference to IEEE 802 here as well, please?

Reply:
Added